funcs_utils.php

```diff
@@ -337,16 +337,6 @@ function current_branch_cms_major(
 ) {
     global $MODULE_DIR, $GITHUB_REF;
 
-    // This repo matched multiple majors and matches start at the lowest - but we only want it for its highest supported one.
-    if ($GITHUB_REF === 'silverstripe/silverstripe-simple') {
-        return '5';
-    }
```

Comment:
No longer included in multiple supported majors, so `5` is the only thing that will match